manifests/server/config.pp

```diff
@@ -122,7 +122,7 @@
 
     file { $dbpath:
       ensure   => directory,
-      mode     => '0755',
+      mode     => '0750',
```

Comment:
Could this be `0640` and `recurse => true` to get rid of the exec below?

Reply:
The exec is hidden behind a boolean. Maybe, but I don't know for sure.